CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## unreleased
 
+braking change to `REGEX_NGINX_COMBINED_LOG`
```

Comment:
please shorten this to just a short single line. A link to the PR can be included for a more detailed explanation.

Reply:
agreed